Issue #1504 - properly process local references in transaction bundles (#1560)
IBM/FHIR: docs/src/pages/guides/FHIRServerUsersGuide.md

#### 4.8.1.2 Example 3: Encounter and Procedure circular references
```
{
    "resourceType" : "Bundle",
    "type" : "batch",
    "entry" : [ {
        "fullUrl" : "urn:Encounter_1",
        "resource" : {
            "resourceType" : "Encounter",
            …
            "reasonReference" : [ {
                    "reference" : "urn:Procedure_1"
            } ],
            …
        },
        "request" : {
            "method" : "POST",
            "url" : "Encounter"
        }
    }, {
        "fullUrl" : "urn:Procedure_1",
        "resource" : {
            "resourceType" : "Procedure",
            …
            "encounter" : {
                    "reference" : "urn:Encounter_1"
            },
            …
        },
        "request" : {
            "method" : "POST",
            "url" : "Procedure"
        }
    } ]
}
```

> While processing a request bundle, but before processing individual request entries, the FHIR server will detect the use of a local identifier within any `POST` or `PUT` request entry's `fullUrl` field, and will establish a mapping between that local identifier and the corresponding external identifier that will result from performing the `POST` or `PUT` operation. 

Reviewer: use an active tense 
```suggestion
While processing a request bundle, but before processing individual request entries, the IBM FHIR Server detects the use of a local identifier within any `POST` or `PUT` request entry's `fullUrl` field, and establishes a mapping between that local identifier and the corresponding external identifier that resultsfrom performing the `POST` or `PUT` operation. 
```

Author: Done - took your suggested changes.
